src/tokens.scss

```diff
@@ -0,0 +1,6 @@
+:host {
+  --ds-auro-menu-border-color: var(--ds-color-border-divider-default, $ds-color-border-divider-default);
+  --ds-auro-menuoption-background-color: transparent;
+  --ds-auro-menuoption-checkmark-color: transparent;
```

Comment:
Let's call this `icon` instead of `checkmark` to future proof against any possible change that supports a different look.